dizoo/slime_volley/config/slime_volley_league_ppo_config.py

```diff
@@ -23,43 +23,28 @@
         ),
         learn=dict(
             epoch_per_collect=5,
-            batch_size=64,
+            batch_size=256,
             learning_rate=3e-4,
             value_weight=0.5,
-            entropy_weight=0.0,
+            entropy_weight=0.005,
             clip_ratio=0.2,
         ),
         collect=dict(
-            n_episode=32,
+            n_episode=16,
             unroll_len=1,
             discount_factor=0.99,
             gae_lambda=0.95,
         ),
         other=dict(
             league=dict(
                 player_category=['default'],
-                path_policy="slime_volley_league_ppo/policy",
-                active_players=dict(main_player=1,
-                                    # main_exploiter=1,
-                                    # league_exploiter=1,
-                                    ),
+                path_policy="slime_volley_league_ppo_base/policy",
+                active_players=dict(main_player=1, ),
                 main_player=dict(
-                    one_phase_step=2000,
-                    branch_probs=dict(pfsp=1.0, ),
+                    one_phase_step=20000,
+                    branch_probs=dict(pfsp=0.2, sp=0.8),
                     strong_win_rate=0.7,
                 ),
-                main_exploiter=dict(
```

Comment:
Why `main_exploiter` and `league_exploiter` were removed here? Could you keep them?

Reply:
I'll first tune current league training with only main player (pfsp + sp) to acquire reasonable performance, then add these two players to construct more meaningful league.